### PR TITLE
fix main readme images background

### PR DESCRIPTION
**Pull request by yrtimiD**
Added white rectangle behind the image to ensure always readable text + some visual padding around. Additionally, this ensures that after future editing of the diagrams on the draw.io bg customizations are not lost.

### CODE REVIEW

File: layers.drawio.svg
Comment (stevenhartley) [inline layers.drawio.svg:1]: Some of the icons have disappeared when I look at the diff (AMQP, DDS, CloudEvent, etc))

File: overview.drawio.svg
Comment (stevenhartley) [inline overview.drawio.svg:1]: Same comment as previous, cloudevent icon is missing.

File: ues.drawio.svg
Comment (stevenhartley) [inline ues.drawio.svg:1]: Font has changed, I would prefer to keep the old font

**Review by stevenhartley — CHANGES_REQUESTED** [review]
Please see comments.

File: overview.drawio.svg
Comment (yrtimiD) [reply overview.drawio.svg:1]: Fixed this. These two images were not originally embedded but fetched from the remote location making them disappear randomly. Now they are embedded and should not make surprises.

File: ues.drawio.svg
Comment (yrtimiD) [reply ues.drawio.svg:1]: thanks, embedded the font, now should work always
(and changed font style for the "Legend" label to be as all other, hope this is ok?)

File: layers.drawio.svg
Comment (yrtimiD) [reply layers.drawio.svg:1]: Fixed this. These two images were not originally embedded but fetched from the remote location making them disappear randomly. Now they are embedded and should not make surprises.

File: overview.drawio.svg
Comment (yrtimiD) [reply overview.drawio.svg:1]: fixed

**Review by stevenhartley — APPROVED** [review]
Thank you for the change!

**Pull request merged**